corehq/form_processor/test_utils.py

```diff
@@ -17,8 +17,8 @@ class FormProcessorTestUtils(object):
     @classmethod
     @unit_testing_only
     def delete_all_cases(cls, domain=None):
+        assert CommCareCase.get_db().dbname.endswith('test')
```

Comment:
heh, in one of my intermediate attempts at this it ran against my real local database and deleted several thousand cases before i realized what was happening